recipe/meta.yaml

```diff
@@ -53,3 +55,4 @@ extra:
     - rhattersley
     - ocefpaf
     - bjlittle
+    - dkillick
```

Comment:
@dkillick this will give you commit rights to this feedstock.

Reply:
Good stuff on this @dkillick!

As ever, thanks @ocefpaf!